registration-processor/core-processor/registration-processor-demo-dedupe-stage/src/test/resources/RegistrationProcessorIdentity.json

```diff
@@ -12,13 +12,13 @@
 			"value": "dateOfBirth",
 			"isMandatory": true
 		},
-		"parentOrGuardianRID": {
+		"introducerRID": {
 			"value": "parentOrGuardianRID"
```

Comment:
since all the RegistrationProcessorIndentity.json files in the code are test files, we can change the value also to introducerRID, introducerUIN etc